zerver/lib/test_classes.py

```diff
@@ -304,29 +297,39 @@ def client_patch(
         self,
         url: str,
         info: Dict[str, Any] = {},
+        skip_user_agent: bool = False,
+        follow: bool = False,
+        secure: bool = False,
         intentionally_undocumented: bool = False,
-        **kwargs: ClientArg,
+        **headers: str,
```

Comment:
`headers` is misleading, since this has different keys from `HttpRequest.headers` (`HTTP_USER_AGENT`, not `User-Agent`) and has more than just HTTP headers (like `REMOTE_ADDR`). Django calls it `extra`.